lib/instrumentation/index.js

```diff
@@ -83,7 +83,7 @@ Instrumentation.prototype.currTransaction = function () {
   if (!this._started) {
     return null
   }
-  return this._runCtxMgr.active().currTransaction() || null
+  return this._runCtxMgr.active().currTransaction()
```

Comment:
REVIEW NOTE: This was redundant. `RunContext.currTransaction()` already defaults to `null` if there isn't a transaction.